### PR TITLE
Add featured location cards to initial app view

### PR DESCRIPTION
**Pull request by danford1**
### Overview
Some initial user feedback revealed that featured location cards may help a user that is not familiar with NJ zoom to a location quickly.

This adds featured location cards to the sidebar in the initial app state that fly-to specific areas.

### Testing
* git pull
* run localhost
* Go to http://localhost:8000/#
* Click on featured location title to zoom
* Draw area of interest
* Click `Draw New Area`
* Zoom in an out to see app state changes

### CODE REVIEW

File: src/app/index.html [hunk 1]
```diff
@@ -555,14 +550,22 @@ <h4>Future possibilities</h4>
...
     setLoading();
   }
+  return;
 }
 
 function setReady() {
-  //console.log('ready');
 }
 
 function setLoading() {
```
Comment (lossyrob) [inline src/app/index.html]: are these unused? if so delete

File: src/app/index.html [hunk 2]
```diff
@@ -621,12 +625,12 @@ <h4>Draw your Area of Interest.</h4> \
       <!-- App State 1 --> \
       </div>';
 
-  // set map canvas equal to div.#map width
-  var mapCanvas = document.getElementsByClassName('mapboxgl-canvas')[0];
-  var mapDiv = document.getElementById('map');
-  mapDiv.style.width = '100%';
-  mapCanvas.style.width = '100%';
-  MAP.resize();
+  // // set map canvas equal to div.#map width
```
Comment (lossyrob) [inline src/app/index.html]: delete commented

File: src/app/index.html [hunk 3]
```diff
@@ -705,15 +709,16 @@ <h4>Area of Interest Habitat List</h4> \
       DRAW.deleteAll();
       unconstrainMap();
       APP.areaOfInterest = emptyFeatureCollection;
-      renderSelectAreaOfInterestPrompt();
+      // renderSelectAreaOfInterestPrompt();
```
Comment (lossyrob) [inline src/app/index.html]: delete commented

File: src/app/index.html [hunk 4]
```diff
@@ -705,15 +709,16 @@ <h4>Area of Interest Habitat List</h4> \
...
+      updateApp();
   }
 
-  // set map canvas equal to div.#map width
-  var mapCanvas = document.getElementsByClassName('mapboxgl-canvas')[0];
-  var mapDiv = document.getElementById('map');
-  mapDiv.style.width = '100%';
-  mapCanvas.style.width = '100%';
-  MAP.resize();
+  // // set map canvas equal to div.#map width
```
Comment (lossyrob) [inline src/app/index.html]: delete commented

**Comment by danford1** [comment]
Thanks @lossyrob. I deleted the unused functions and commented code in the latest commit.

**Pull request merged**